drivers/modem/modem_receiver.c

```diff
@@ -17,7 +17,7 @@
 
 #include <logging/log.h>
 #define LOG_DOMAIN mdm_receiver
-#define LOG_LEVEL CONFIG_LOG_MODEM_LEVEL
+#define LOG_LEVEL CONFIG_MODEM_LOG_LEVEL
 LOG_MODULE_REGISTER(LOG_DOMAIN);
```

Comment:
those 3 lines may be converted to:
`LOG_MODULE_REGISTER(mdm_receiver, CONFIG_MODEM_LOG_LEVEL);`
this has the advantage that not existing defines would be reported.

Reply:
Fixed